packages/ui/components/editor/Editor.tsx

```diff
@@ -74,19 +76,22 @@ export const Editor = (props: TextEditorProps) => {
     <div className="editor rounded-md">
       <LexicalComposer initialConfig={{ ...editorConfig }}>
         <div className="editor-container hover:border-emphasis focus-within:ring-brand-default rounded-md p-0 transition focus-within:ring-2">
-          <ToolbarPlugin
-            getText={props.getText}
-            setText={props.setText}
-            editable={editable}
-            excludedToolbarItems={props.excludedToolbarItems}
-            variables={props.variables}
-            updateTemplate={props.updateTemplate}
-            firstRender={props.firstRender}
-            setFirstRender={props.setFirstRender}
-          />
           <div
             className={classNames("editor-inner scroll-bar overflow-x-hidden", !editable && "!bg-subtle")}
             style={{ height: props.height, maxHeight: props.maxHeight }}>
+            {props.action !== WorkflowActions.SMS_ATTENDEE && props.action !== WorkflowActions.SMS_NUMBER && (
+              <ToolbarPlugin
+                getText={props.getText}
+                setText={props.setText}
+                editable={editable}
+                excludedToolbarItems={props.excludedToolbarItems}
+                variables={props.variables}
+                updateTemplate={props.updateTemplate}
+                firstRender={props.firstRender}
+                setFirstRender={props.setFirstRender}
+                action={props.action}
+              />
+            )}
```

Comment:
This is not good. Conditionally use a plugin for a particular use case. I think you can use the Editor's `excludedToolbarItems `property. We've used it in other components as well

Reply:
Are we talking about something like this?
```
excludedToolbarItems={
       step.action !== WorkflowActions.SMS_ATTENDEE && step.action !== WorkflowActions.SMS_NUMBER
           ? []
       : ["blockType", "bold", "italic"]
 }

Reply:
Also these conditions were there in `WorkflowStepContainer`.

Reply:
> Are we talking about something like this?

You can try